api/HTMLScriptElement.json

```diff
@@ -201,7 +201,7 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.5"
+              "version_added": "1"
```

Comment:
There's a "defer" entry in script.json with notes, updating this would require a bit more care. I'll revert.